### DISCUSSION TITLE
Facing issue in building iOS build with CodeMagic CI/CD tool

### DISCUSSION THREAD

**Discussion by akjainplotx** [codemagic-ci-cd/codemagic-docs#1384] [Q&A]
**Current configuration**
**Flutter channel**: default
**Mode**: release
**Xcode version**: 13.4.1
**Build for**: iOS

Hi All, 
I am facing issue while creating build for IOS. I was able to **build artifact successfully** using same git **commit** previously. 
But now, this is getting failed right now. 

Also, same build is getting built successfully on local machine. 

ERROR LOG : 
Error (Xcode): 142 duplicate symbols for architecture arm64
Encountered error while archiving for device.

**Comment by icarusdust** [comment]
Hi @akjainplotx , as you raised the issue in Intercom, let's continue there.
